CoThings/ViewController.swift

```diff
@@ -136,18 +136,11 @@ class ViewController: UIViewController, CLLocationManagerDelegate, UITableViewDe
             .delegateReceive("ok", to: self) { (self, data) in
                 self.infoLabel.text =  "Socket: Joined Channel"
                 
-                if let dictionary = data.payload["response"] as? [String: Any] {
-                    if let array = dictionary["rooms"] as? [Any] {
-                        
-                        for object in array {
-                            let room = object as? [String: Any]
-                            let name = room?["name"] as? String
-                            self.rooms.append(name!)
-                        }
-                        
-                        self.lobbyTable.reloadData()
-                    }
+                guard let response = MessageAdapter(message: data).decodeRoomResponse() else {
+                    return
```

Comment:
Didn't check the whole code, but.. Do we have any way of error handling instead of ignoring? (discussion)

Reply:
We don't have error handling mechanism but better to have at the beginning, I'll also ask kindly to designer friends to show error cases in the app.

Reply:
Until we have a general flow of handling errors, we can just silence it here, and create a new issue for handling errors.

In that case it's better to leave a TODO commont here, or an issue number if possible. This would also allow merging this work without getting blocked on this.

Reply:
Going to be handled in this issue => https://github.com/rainlab-inc/cothings-ios/issues/2
resolving.